docs/specs/schema.yml

```diff
@@ -154,6 +155,7 @@ outputs:
     { "content": {} }
   .errors.log: |
     ["info","null-value","'description' contains null value","docs/a.yml",2,1]
+    ["warning","unexpected-type","Expect type 'String' but got 'Null'","docs/a.yml",2,13]
```

Comment:
duplicate with above `null-value`. Should log only one message. Looks like a bug in schema validation.

Reply:
yes, already opened a issue for that: #4591